### PR TITLE
endpoint for row state update

### PR DESCRIPTION
**Pull request by ErikZigo**
PR: https://github.com/keboola/connection/pull/2906

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -5660,6 +5660,90 @@ independently on the configurations.
...
+
++ Response 200 (application/json)
+
+            {
+                "id": "236781529",
+                "configuration": {
+                    "host": "localhost",
+                    "port": null
+                },
+                "state": {},
```
Comment (fejjo) [inline apiary.apib:5691]: tu by v tej ukazke by mohlo byt ten state co mas v tom body v ukazke requestu

File: src/Keboola/StorageApi/Components.php [hunk 2]
```diff
@@ -83,11 +76,19 @@ public function updateConfiguration(Configuration $options)
 
     public function updateConfigurationState(ConfigurationState $options)
     {
+        $data = [];
+
+        if ($options->getState() !== null) {
+            if ($options->getState() === []) {
+                $data['state'] = '{}';
```
Comment (fejjo) [inline src/Keboola/StorageApi/Components.php:83]: ❤️

**Review by fejjo — APPROVED** [review]
Cajk, asi by som upravil tu dokumentaciu

File: apiary.apib (hunk 1 above)
Comment (ErikZigo) [reply apiary.apib]: Upravil jsem to i pro konfigurace.

**Pull request merged**